manager/state/raft/raft_test.go

```diff
@@ -226,7 +227,9 @@ func TestRaftQuorumRecovery(t *testing.T) {
 	// Restore the majority by restarting node 3
 	nodes[3] = raftutils.RestartNode(t, clockSource, nodes[3], false)
 
+	raftutils.ShutdownNode(nodes[1])
 	delete(nodes, 1)
+	raftutils.ShutdownNode(nodes[2])
```

Comment:
Not sure this is right. How can `WaitForCluster` succeed if the two out of three members are shut down? I think this should be after the `WaitForCluster`.